addons/web/static/src/core/bottom_sheet/bottom_sheet.scss

```diff
@@ -274,12 +319,23 @@
 
         font-size: $h5-font-size;
         font-weight: $o-font-weight-medium;
+        text-align: start !important;
     }
 
     .o_bottom_sheet_body:not(.o_custom_bottom_sheet) {
         // Dropdown
         .dropdown-item {
-            &.active, &.selected {
+            &.selected {
+                @extend %BottomSheet-Entry-active;
+                @extend %BottomSheet-Entry-selected;
```

Comment:
Both states? Sounds wrong... it may be the cause of this issue (calendar view)
<img width="465" height="492" alt="image" src="https://github.com/user-attachments/assets/2c4093d2-e581-4ae6-bcb3-66f3ad44dac8" />

Reply:
Good catch 👍 
The issue comes to [my selector](https://github.com/odoo/odoo/pull/227230/files#diff-d0207dbfa92b174873366c34ae08ac9a10f6c387c5534d7ce33f4c1bb7170237R335-R336) that was not strong enough to target the "Progress" items, and so the style is also applied when it should not, like here in calendar.
Unfortunately, we should need a flag class to trigger this design correctly. 🥲
As we can't do this in stable, I'll revert this change.

Reply:
Also there is the `project_task_state_selection_menu` dropdown that is impacted
<img width="360" height="626" alt="image" src="https://github.com/user-attachments/assets/9d8422d6-eac3-48f5-b92d-7d877b05eaad" />